.github/workflows/bb_containers.yml

```diff
@@ -76,37 +62,43 @@ jobs:
             image: ubuntu:20.04
             branch: 10.7
             platforms: linux/amd64, linux/arm64/v8, linux/ppc64le, linux/s390x
-          - dockerfile: debian.Dockerfile
-            image: ubuntu:21.10
-            branch: 10.7
-            platforms: linux/amd64, linux/arm64/v8, linux/ppc64le, linux/s390x
           - dockerfile: debian.Dockerfile
             image: ubuntu:22.04
             branch: 10.7
             platforms: linux/amd64, linux/arm64/v8, linux/ppc64le, linux/s390x
-          - dockerfile: fedora.Dockerfile
-            image: fedora:34
-            platforms: linux/amd64, linux/arm64/v8
           - dockerfile: fedora.Dockerfile
             image: fedora:35
             platforms: linux/amd64, linux/arm64/v8
           - dockerfile: fedora.Dockerfile
             image: fedora:36
             platforms: linux/amd64, linux/arm64/v8
+# conflicts - see https://github.com/MariaDB/buildbot/runs/7106820431?check_suite_focus=true#step:11:2205
+#          - dockerfile: fedora.Dockerfile
+#            image: fedora:37
+#            platforms: linux/amd64, linux/arm64/v8
           - dockerfile: centos7.Dockerfile pip.Dockerfile
             image: centos:7
-            platforms: linux/amd64, linux/arm64/v8, linux/ppc64le
+            platforms: linux/amd64
           - dockerfile: centos.Dockerfile
             image: quay.io/centos/centos:stream8
             tag: centos-stream8
-            platforms: linux/amd64, linux/arm64/v8
+            platforms: linux/amd64, linux/arm64/v8, linux/ppc64le
```

Comment:
ppc64le-centos-stream8 is not enabled anywhere in the constants.py. Do we want to add it?